docker/nestml_docker.sh

```diff
@@ -75,7 +75,7 @@ case $mode in
 	cmd="docker build -t $container -f $dockerfile"
```

Comment:
@jougs 

```
nestml@nestml:~/software/nestml/docker$ ./nestml_docker.sh run ../models/

Running docker image 'nestml_release'
Error: Unable to access jarfile /data/nestml/target/nestml.jar

nestml@nestml:~/software/nestml/docker$ ./nestml_docker.sh run --dev ../models/

Running docker image 'nestml_development'
Error: Unable to access jarfile /data/nestml.jar
```
Entrypoint script should either distinguish between development and release versions correclty or the nestml.jar should be stored at the same place (which can be done in corresponding  Dockerfile-artifacts)